sale_payment_mgmt/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,17 @@
+This addon adds a Payments (only customer payments) menu in Sales to users are in some of this Sale groups:
+
+    User: Own Documents Only
+    User: All Documents
+    Administrator
+
+.. image:: ../static/description/payment_menu.png
+
+Show "Register payment" button in invoices (that allow more faster create and validate payment related) to users in some of this Sale groups:
+
+    User: Own Documents Only
+    User: All Documents
+    Administrator
+
+Hide 'Vendor' option in Partner type about create payments if user not in Invoicing group
```

Comment:
No need to mention this, as it's already mentioned before that only customer payments.